clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAOSpec.scala

```diff
@@ -112,41 +105,49 @@ class HttpElasticsearchDAOSpec
         val error = exception.requestFailure.error
         error.`type` should be("illegal_argument_exception")
         error.reason should be(
-          "mapper [foo] of different type, current_type [keyword], merged_type [long]"
+          "mapper [mock_string_array] of different type, current_type [keyword], merged_type [text]"
         )
       }
     } yield succeed
   }
 
   it should "return the most recent document" in {
-    import HttpElasticsearchDAOSpec._
     import org.broadinstitute.clio.server.dataaccess.elasticsearch._
 
-    val index = new ElasticsearchIndex[Document](
-      "docs-" + UUID.randomUUID(),
+    val index = new ElasticsearchIndex[ModelMockIndex](
+      ModelMockIndex("docs-" + UUID.randomUUID()),
       ElasticsearchFieldMapper.NumericBooleanDateAndKeywordFields
     )
 
     val documents =
       (1 to 4)
         .map("document-" + _)
-        .map(s => Document(UpsertId.nextId(), Symbol(s)))
+        .map(
```

Comment:
Oof... this should get slightly less wonky if you use block-form for the arg to `map`, and build the bookkeeping as a single `Map`